File: FCUBEAPI/Consignment/Models/DoVehiPlacedListModel.cs
using Shared.Models;

namespace Consignment.Models
{
    public class DoVehiPlacedListModel
    {
        public List<DoVehiPlacedModel> DoVehiPlacedList { get; set; }
        public PaginationMetaData PageMetaData { get; set; }
    }

}

File: OMKARAUTOAPP/src/app/models/dovehiplacedlistmodel.ts
import { Paginationmodel } from "./paginationmodel";
import { Dovehiplacedmodel } from "./dovehiplacedmodel";

export class Dovehiplacedlistmodel {
    doVehiPlacedList: Dovehiplacedmodel[] = [];
    pageMetaData: Paginationmodel = new Paginationmodel;
}
